tests/workgroup/CMakeLists.txt

```diff
@@ -62,6 +62,10 @@ add_test_pocl(NAME "workgroup/cond_barriers_in_for"
               EXPECTED_OUTPUT "cond_barriers_in_for_2_4_1_1.stdout"
               COMMAND "run_kernel" "cond_barriers_in_for.cl" 2 4 1 1)
 
+add_test_pocl(NAME "workgroup/cond_barrier_in_var_for"
```

Comment:
This should be ran only with the CBS, but the rest (that work with both) both with wiloops and cbs, right?